src/database/seed.sql

```diff
@@ -0,0 +1,7 @@
+\c todos
+
+INSERT INTO todos (name, description) VALUES
+('Groceries', '10 items')
+('Clean Room', 'Storage')
+('Meditate', '20Min')
+('Jobs', 'Apply to 20')
```

Comment:
The table above only has a description field not a name, so either change the schema or remove one of the fields from the seed data